setup.py

```diff
@@ -64,4 +65,7 @@ def find_version(*file_paths):
     python_requires=">=3",
     install_requires=install_requires,
     packages=find_packages(),
+    entry_points = {
+        "console_scripts": ["deepo=deeposlandia.__main__:main"],
```

Comment:
I prefer to keep the name short for a sake of usability (instead of aliasing it afterwards... that's already an alias, after all), if it ain't bother you I will let it as is.